Library/Formula/objective-caml.rb

```diff
@@ -18,20 +19,18 @@ class ObjectiveCaml < Formula
   end
 
   def install
-    args = %W[
-      --prefix #{HOMEBREW_PREFIX}
-      --mandir #{man}
-      -with-debug-runtime
-    ]
-    args << "-cc" << "#{ENV.cc} #{ENV.cflags}"
-    args << "-aspp" << "#{ENV.cc} #{ENV.cflags} -c"
-    args << "-no-graph" if build.without? "x11"
-
     ENV.deparallelize # Builds are not parallel-safe, esp. with many cores
+
+    # the ./configure in this package is NOT a GNU autoconf script!
+    args = ["-prefix", prefix, "-with-debug-runtime", "-mandir", man]
```

Comment:
@avsm If you could also add a test to the `test do` to reproduce/verify this that'd be great, thanks!